package.json

```diff
@@ -34,6 +34,9 @@
     "formik": "^2.2.9",
     "framer-motion": "^7.3.6",
     "history": "^5.3.0",
+    "i18next": "^23.11.5",
+    "i18next-fs-backend": "^2.3.1",
```

Comment:
Maybe I missed it but it does not seem to be used ?

Reply:
Leftover from testing. I'll remove it.